fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -175,6 +175,15 @@ def run
       # @!group Plugins
       #####################################################
 
+      command :create_plugin do |c|
```

Comment:
In the docs I called the command `new_plugin`: https://github.com/fastlane/fastlane/pull/4722/files#diff-cf9ca7bd4b992dfc6d9b6cdb37a890c0R54

We can do either way, however similar tools like danger use `new` instead of `create`: https://github.com/danger/danger/blob/18d5875ab2da882b068e7d2356d1f444a82dfaed/lib/danger/commands/new_plugin.rb#L4